robustness/attacker.py

```diff
@@ -266,7 +266,7 @@ class AttackerModel(ch.nn.Module):
 
     More code examples available in the documentation for `forward`.
     For a more comprehensive overview of this class, see `our detailed
-    walkthrough <../example_usage/input_space_manipulation>`_
+    walkthrough <../example_usage/input_space_manipulation.rst>`_
```

Comment:
Weirdly enough I think it works without the .rst (https://robustness.readthedocs.io/en/latest/api/robustness.attacker.html), maybe worth leaving it?

Reply:
Fixed this, see 59106e8c4dc5919de49e31e8782367b32bc24127